lyncs_quda/lattice_field.py

```diff
@@ -258,8 +254,46 @@ def prepare_in(self, fields, **kwargs):
         kwargs.setdefault("copy", True)
         return self.prepare(fields, **kwargs)
 
+    _children = {}
+    
+    def __new__(cls, field, **kwargs):
+        #TODO: get dofs and local dims from kwargs, instead of getting them
+        # from self.shape assuming that it has the form (dofs, local_dims)
+
+        if isinstance(field, LatticeField):
```

Comment:
I would check if instance of `cls`, because being instance of `LatticeField`, does not mean being instance of `cls`.